Detectors/PHOS/reconstruction/src/AltroDecoder.cxx

```diff
@@ -312,6 +312,11 @@ void AltroDecoder::readTRUFlags(short hwAddress, int payloadSize)
       return;
     }
     int timeBin = mBunchwords[currentsample + 1] + 1; // +1 for further convenience
+    if (timeBin > 120) {                              // corrupted sample: add error and ignore the reast of bunchwords
```

Comment:
Size of bitsets below is 128. Why did you put check here 120 and not 128?
Otherwise I agree with this commit

Reply:
Hi @peressounko 
from the first look I thought it should be 128 but then I tried to get the idea of the code and came to conclusion that values >= 120 are not allowed by design. It seems it is wrong conclusion though. Can timeBin be > 120? Is it limited by 128? If so then I change.

Reply:
Hi @sevdokim, I do not see where it is limited by 120. In any case 120/128 is redundant as it corresponds to times 12.x mks, what is much larger than PHOS readout time (~2 mks) and will never be used in analysis. However, I would change to 128 just for readability of the code.